Propose a simpler Quickstart (#256)
elastic/crawler: docs/CLI.md

## Installation and Configuration

Ensure you are able to run Crawler in docker or have completed the steps to run Crawler [from source](../README.md#running-open-crawler-from-source) before using the CLI.

For instructions on configuring a Crawler, see [CONFIG.md](./CONFIG.md).

### CLI in Docker

To run individual CLI commands in Docker, you can use the following format

```bash
docker run -it docker.elastic.co/integrations/crawler:latest jruby \
    bin/crawler <command> <args>
```

For an interactive shell with Crawler, simply change the entrypoint to `/bin/bash`:

```bash
docker run -it --entrypoint /bin/bash docker.elastic.co/integrations/crawler:latest
```

If you need to mount a file into the container, for example `crawl-config.yml`, you can use the `-v` option to mount it into the container:

```bash

> docker run -it -v ./crawl-config.yml:/crawl-config.yml docker.elastic.co/integrations/crawler:latest ruby \

Author: ```suggestion
docker run -it -v ./crawl-config.yml:/crawl-config.yml docker.elastic.co/integrations/crawler:latest jruby \
```

## Installation and Configuration

Ensure you are able to run Crawler in docker or have completed the steps to run Crawler [from source](../README.md#running-open-crawler-from-source) before using the CLI.

For instructions on configuring a Crawler, see [CONFIG.md](./CONFIG.md).

### CLI in Docker

To run individual CLI commands in Docker, you can use the following format

```bash

> docker run -it docker.elastic.co/integrations/crawler:latest ruby \
>     bin/crawler <command> <args>

Reviewer: Make sure we aren't invoking C-based ruby

```suggestion
docker run -it docker.elastic.co/integrations/crawler:latest jruby \
    bin/crawler <command> <args>
```

```

For an interactive shell with Crawler, simply change the entrypoint to `/bin/bash`:

```bash
docker run -it --entrypoint /bin/bash docker.elastic.co/integrations/crawler:latest
```

> If you need to mount a file into the container, for example `crawl.yml`, you can use the `-v` option to mount it into the container:
> 
> ```bash
> docker run -it -v ./crawl.yml:/crawl.yml docker.elastic.co/integrations/crawler:latest ruby \
>     bin/crawler crawl /crawl.yml
> ```

Reviewer: ```suggestion
If you need to mount a file into the container, for example `crawl-config.yml`, you can use the `-v` option to mount it into the container:

```bash
docker run -it -v ./crawl-config.yml:/crawl-config.yml docker.elastic.co/integrations/crawler:latest ruby \
    bin/crawler crawl /crawl-config.yml
```
```
